epam-ios-guess-the-number/epam-ios-guess-the-number/ViewController.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  ViewController.swift
+//  epam-ios-guess-the-number
+//
+//  Created by Артем Маков on 23/01/2020.
+//  Copyright © 2020 Артем Маков. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController, UITextFieldDelegate {
+    
+    @IBOutlet weak var label: UILabel!
+    @IBOutlet weak var button: UIButton!
+    @IBOutlet weak var inputField: UITextField!
+    @IBOutlet weak var ansLabel: UILabel!
+    var imagineNumber = 0;
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(true)
+        imagineNumber = Int.random(in: 0 ..< 100)
+        button.isEnabled = false
+        inputField.delegate = self
+        inputField.keyboardType = .numberPad
+        self.ansLabel.text = ""
+        //print(imagineNumber)
+    }
+    
+    func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
+        // Find out what the text field will be after adding the current edit
+        let text = (textField.text! as NSString).replacingCharacters(in: range, with: string)
```

Comment:
Please don't use force unwrap.
if let, guard let, ??
Are NSString required here ?
Swift has it's own String.